scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -102,28 +106,55 @@
       editor.getSession().setAnnotations(annotations);
     }
 
-    const $selectionButtons = getAll('.tabs a');
+    let valueIndex = 0;
+    const selectionButtons = getAll('.tabs a');
+    const prevBtn = document.querySelector('.prev-btn');
+    const nextBtn = document.querySelector('.next-btn');
 
     let detected_values = JSON.parse(document.querySelector("#detected_values").textContent);
+    let detected_data;
 
-    if ($selectionButtons.length > 0) {
-      $selectionButtons.forEach(function ($el) {
+    if (selectionButtons.length > 0) {
+      selectionButtons.forEach(function ($el) {
         $el.addEventListener('click', function () {
+          removeAllMarkers();
           let active_li = document.querySelector("li.is-active");
           if (active_li) active_li.classList.remove("is-active");
-          $el.parentElement.classList.add("is-active");
 
+          $el.parentElement.classList.add("is-active");
           let type = $el.getAttribute("data-type");
-          let detected_data = detected_values[type];
-          if (detected_data) {
+          detected_data = detected_values[type];
+
+          if (detected_data.length) {
+            valueIndex = 0;
             setDetectedValues(detected_data);
+            editor.renderer.scrollToLine(detected_data[0].start_line - 1);
+            prevBtn.disabled = true;
+            nextBtn.disabled = false;
+          } else {
+            valueIndex = 0;
+            editor.renderer.scrollToLine(0);
+            prevBtn.disabled = true;
+            nextBtn.disabled = true;
           }
-          else {
-            removeAllMarkers();
-            editor.getSession().setAnnotations([]);
-          }
-        });
+      });
       });
     }
+
+    function nextValue() {
+        if (valueIndex >= detected_data.length - 1) return false;
+        valueIndex++;
+        editor.renderer.scrollToLine(detected_data[valueIndex].start_line - 1);
+        nextBtn.disabled = (valueIndex == detected_data.length - 1) ? true : false; 
+        prevBtn.disabled = false;
+
+    }
+    function previousValue() {
+        if (valueIndex <= 0) return false;
+        valueIndex--;
+        editor.renderer.scrollToLine(detected_data[valueIndex].start_line - 1);
+        prevBtn.disabled  = (valueIndex == 0) ? true : false;
```

Comment:
Can be simplified to `valueIndex == 0`.

Reply:
`previousBtn.disabled = (scrollPositionIndex == 0);` like this?

Reply:
Fixed this.